src/screens/ContactScreen/index.js

```diff
@@ -212,11 +227,16 @@ const ContactScreen = ({navigation}) => {
         // onPress={handleSearch}
       />
 
-      <View>
-        {selectedUsers && (
-          <ContactPreview users={selectedUsers} onPress={(user) => handleSelected(user)} />
-        )}
-      </View>
+      {sourceScreen !== ANONYMOUS && (
+        <View>
+          {selectedUsers && (
+            <ContactPreview
+              users={selectedUsers}
+              onPress={(user) => handleSelected(user, sourceScreen)}
+            />
+          )}
+        </View>
+      )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `ContactPreview` component is now only rendered if the source screen is not `ANONYMOUS`. This seems like a reasonable change, but make sure that this doesn't negatively affect any other parts of the application that rely on this component being rendered.



<!-- This is an auto-generated comment by OSS CodeRabbit -->